doc/source/tutorials/local/Adding_new_data/Select_write_store.rst

```diff
@@ -0,0 +1,286 @@
+Select a store to write to
+==========================
+
+This tutorial assumes you have write priviliges to multiple object stores and for the purposes of the
+tutorial we'll create a new config file with multiple stores to detail this process.
+
+0. Backing up your config file
+------------------------------
+
+To avoid deletion of your current config file we'll first create a backup
+
+.. code:: bash
+
+    $ cp ~/.config/openghg/openghg.conf $ cp ~/.config/openghg/openghg.conf.bak
+
+
+1. Create a new config
+-----------------------
+
+We'll now run ``openghg --quickstart`` and create a new configuration file with two object stores,
+both in our home directory.
+
+.. code-block:: bash
+
+    $ openghg --quickstart
+
+    OpenGHG configuration
+    ---------------------
+
+    INFO:openghg.util:We'll first create your user object store.
+
+    Enter path for your local object store (default /home/gareth/openghg_store):
+    Would you like to add another object store? (y/n): y
+    Enter the name of the store: shared
+    Enter the object store path: /home/gareth/openghg_shared
+
+    You will now be asked for read/write permissions for the store.
+    For read only enter r, for read and write enter rw.
+
+    Enter object store permissions: rw
+    Would you like to add another object store? (y/n): n
```

Comment:
Does this allow more than 2 object stores to be added? Would be nice if this was general for instance.